local_planner/include/local_planner/local_planner.h

```diff
@@ -57,8 +57,8 @@ class LocalPlanner {
   int reproj_age_;
   int counter_close_points_backoff_ = 0;
 
-  float curr_yaw_fcu_frame_;
-  float curr_pitch_fcu_frame_;
+  float curr_yaw_deg_fcu_frame_, curr_yaw_deg_histogram_frame_;
```

Comment:
Purely a matter of taste, but I find it a big strange that pitch doesn't specify the frame while yaw does. (I know that there is no difference between the two frames for pitch, but not everyone might know that). Also, I prefer the unit to be specified last, so it is more consistent with naming of things that don't have a specified frame (e.g. `back_off_dist_m_` or something like that). With time, you'll get used to variables having the unit specified in the end